lib/shared.h

```diff
@@ -0,0 +1,28 @@
+#include <stdlib.h>
```

Comment:
Hi, you are right, we will add "marshal.h" rather than "shared.h" to server site (server-kv & its run/test cases) and eliminate "shared.h"